### PR TITLE
moar: init at 1.10.0

### PR DESCRIPTION
**Pull request by foo-dogsquared**
###### Description of changes

Add [moar](https://github.com/walles/moar), a nice pager to use without configuration.

###### Things done

- Built on platform(s)
  - [x] x86_64-linux
  - [ ] aarch64-linux
  - [ ] x86_64-darwin
  - [ ] aarch64-darwin
- [ ] For non-Linux: Is `sandbox = true` set in `nix.conf`? (See [Nix manual](https://nixos.org/manual/nix/stable/command-ref/conf-file.html))
- [ ] Tested, as applicable:
  - [NixOS test(s)](https://nixos.org/manual/nixos/unstable/index.html#sec-nixos-tests) (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
  - and/or [package tests](https://nixos.org/manual/nixpkgs/unstable/#sec-package-tests)
  - or, for functions and "core" functionality, tests in [lib/tests](https://github.com/NixOS/nixpkgs/blob/master/lib/tests) or [pkgs/test](https://github.com/NixOS/nixpkgs/blob/master/pkgs/test)
  - made sure NixOS tests are [linked](https://nixos.org/manual/nixpkgs/unstable/#ssec-nixos-tests-linking) to the relevant packages
- [ ] Tested compilation of all packages that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review rev HEAD"`. Note: all changes have to be committed, also see [nixpkgs-review usage](https://github.com/Mic92/nixpkgs-review#usage)
- [x] Tested basic functionality of all binary files (usually in `./result/bin/`)
- [22.11 Release Notes (or backporting 22.05 Release notes)](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md#generating-2211-release-notes)
  - [ ] (Package updates) Added a release notes entry if the change is major or breaking
  - [ ] (Module updates) Added a release notes entry if the change is significant
  - [ ] (Module addition) Added a release notes entry if adding a new NixOS module
  - [ ] (Release notes changes) Ran `nixos/doc/manual/md-to-db.sh` to update generated release notes
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md).

### CODE REVIEW

File: pkgs/tools/misc/moar/default.nix [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+  nativeBuildInputs = [ installShellFiles ];
+
+  postInstall = ''
+    installManPage ./moar.1
+  '';
+
+  meta = with lib; {
+    description = "Nice-to-use pager for humans";
+    homepage = "https://github.com/walles/moar";
+    license = licenses.bsd2; # More specifically, BSD2 with views sentence.
```
Comment (Atemu) [inline pkgs/tools/misc/moar/default.nix:25]: Could you add the license similar to [this](https://github.com/NixOS/nixpkgs/blob/b2444828a1a6682a3976d8ab9758b26bca9a7592/lib/licenses.nix#L152-L155)?

**Review by Atemu — APPROVED** [review]
Result of `nixpkgs-review pr 201280` run on aarch64-darwin [1](https://github.com/Mic92/nixpkgs-review)
<details>
  <summary>1 package built:</summary>
  <ul>
    <li>moar</li>
  </ul>
</details>

Viewed a file with moar.

It's located in the same directory as `bat` which sound good to me.

File: pkgs/tools/misc/moar/default.nix (hunk 1 above)
Comment (foo-dogsquared) [reply pkgs/tools/misc/moar/default.nix:25]: Sure thing! Does it have to be in a different PR or this PR is enough?

Comment (Atemu) [reply pkgs/tools/misc/moar/default.nix:25]: Just add it in this one; as a separate commit of course.

File: lib/licenses.nix [hunk 2]
```diff
@@ -154,6 +154,11 @@ in mkLicense lset) ({
     fullName = "BSD-2-Clause Plus Patent License";
   };
 
+  bsd2WithViews = {
+    spdxId = "BSD-2-Clause-Views";
+    fullName = "BSD 2-Clause with views sentence";
```
Comment (Atemu) [inline lib/licenses.nix:159]: Both look correct to me

**Review by Atemu — APPROVED** [review]

**Comment by onny** [comment]
Result of `nixpkgs-review pr 201280` run on x86_64-linux [1](https://github.com/Mic92/nixpkgs-review)
<details>
  <summary>2 packages blacklisted:</summary>
  <ul>
    <li>nixos-install-tools</li>
    <li>tests.nixos-functions.nixos-test</li>
  </ul>
</details>
<details>
  <summary>1 package built:</summary>
  <ul>
    <li>moar</li>
  </ul>
</details>

**Comment by onny** [comment]
Thank you!

**Pull request merged**